package.json

```diff
@@ -29,7 +29,7 @@
     "preinstall": "npx check-engine",
     "start": "tsx ./lib/index.ts",
     "start:prod": "npm run build && node ./build/lib",
-    "start:watch": "nodemon tsx ./lib/index.ts",
+    "start:watch": "nodemon --watch 'lib' -e ts --exec \"tsx ./lib/index.ts\"",
```

Comment:
Au lieu d'utiliser `nodemon`, on pourrait sûrement directement utilisé le `watch` mode de `tsx`?

https://tsx.is/watch-mode

Reply:
je vais tester cela :)

Reply:
Ca semble fonctionner @bpetetot !